src/assets/scripts/index.js

```diff
@@ -0,0 +1,63 @@
+import Debug from 'debug';
+
+import peers, {Peer} from './peer';
+import Socket from './socket';
+
+const debug = Debug('app:main');
+const client = {id: null};
+
+const handlers = {
```

Comment:
I prefer static variables that are either primitive types or very simple objects to be capitalized.